pkgs/applications/misc/navi/default.nix

```
@@ -0,0 +1,36 @@
+{ rustPlatform, fetchFromGitHub, stdenv, fzf, makeWrapper, ... }:
```

```suggestion
{ rustPlatform, fetchFromGitHub, stdenv, fzf, makeWrapper }:
```

We don’t use `...` in packages because callPackage passes in only the arguments we need,